testassets/InteropTestsClient/InteropClient.cs

```diff
@@ -768,5 +787,27 @@ private static Metadata CreateTestMetadata()
                 {"x-grpc-test-echo-trailing-bin", new byte[] {0xab, 0xab, 0xab}}
             };
         }
+
+        private byte[] GetBytesFromPem(string pemString, string section)
```

Comment:
looks like this is a hacky way to read the certificate?  use a library function or at least or add a TODO to implement this properly?